src/BDSComponentFactory.cc

```diff
@@ -626,6 +626,15 @@ BDSAcceleratorComponent* BDSComponentFactory::CreateRF(RFFieldDirection directio
   auto modulator = ModulatorDefinition(element);
   vacuumField->SetModulatorInfo(modulator); // works even if none
 
+  auto defaultUL = BDSGlobalConstants::Instance()->DefaultUserLimits();
+  G4double stepFraction  = 0.025;
+  G4double period = 1. / (element->frequency*CLHEP::hertz);
```

Comment:
Here, period is unused (no compiler warning?). 1 is hard coded but if we really need a hard coded constant, then best to put the units in explicitly, e.g. 1*CLHEP::mm.